src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Machine {
+
+    public static void doWork() {
+
+    }
+
+    public static void stopWork() {
```

Comment:
The 'stopWork' method should also have an informative message similar to 'doWork'. As it is now, it does nothing, which is not informative for the subclasses that will inherit this method.